trace-forward/src/Trace/Forward/Forwarder.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE LambdaCase #-}
+
+-- This top-level module will be used by the forwarder application.
+-- Forwarder application collects 'TraceObject's and sends them to
+-- the acceptor application.
+module Trace.Forward.Forwarder
+  ( runTraceForwarder
+  ) where
+
+import qualified Codec.Serialise as CBOR
+import           Control.Concurrent.STM.TBQueue (TBQueue)
+import           Control.Exception (SomeAsyncException (..), fromException, tryJust)
+import           Data.Typeable (Typeable)
+import           System.Time.Extra (sleep)
+
+import           Ouroboros.Network.IOManager (IOManager)
+import           Ouroboros.Network.Util.ShowProxy (ShowProxy(..))
+
+import           Trace.Forward.Configuration (ForwarderConfiguration (..))
+import           Trace.Forward.Network.Forwarder (connectToAcceptor)
+
+runTraceForwarder
+  :: (CBOR.Serialise lo,
+      ShowProxy lo,
+      Typeable lo)
+    => IOManager
+  -> ForwarderConfiguration lo -- ^ Forwarder configuration.
+  -> TBQueue lo                -- ^ The queue from which the forwarder will take 'TraceObject's.
+  -> IO ()
+runTraceForwarder iomgr config loQueue =
+  tryJust excludeAsyncExceptions (connectToAcceptor iomgr config loQueue) >>= \case
+    Left e -> do
+      putStrLn $ "trace-forward, forwarder has a problem: " <> show e
```

Comment:
What about something like this:
```
      putStrLn $ "trace-forward couldn't connect to " <> show address <> ": " <> show e
```
It would be nicer to use a tracer rather than `putStrLn` directly, even if you just trace it to `stdout`.

Reply:
Done.